frontend/src/static/js/v_zoom.js

```diff
@@ -110,13 +109,29 @@ window.vZoom = {
       this.expandedCommitMessagesCount = document.getElementsByClassName('commit-message message-body active')
           .length;
     },
+
+    removeZoomHashes() {
+      window.removeHash('zA');
+      window.removeHash('zR');
+      window.removeHash('zACS');
+      window.removeHash('zS');
+      window.removeHash('zU');
+      window.removeHash('zFGS');
+      window.removeHash('zFTF');
+      window.removeHash('zMG');
+      window.encodeHash();
+    },
   },
   created() {
     this.initiate();
   },
   mounted() {
+    this.setInfoHash();
```

Comment:
The reason why `setInfoHash` is moved to the `mounted` hook is that the `created` hook of the newly created component runs before the `beforeDestroy` of the previously created component.